src/index.html

```diff
@@ -3,8 +3,47 @@
   <head>
     <meta charset="utf-8"/>
     <title>Tax Calculator</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
Remove this line. Styles will be added by webpack. You can inspect a page `dist/index.html` and see that it has a `style` element in `head`